src/components/PermissionPopup/style.js

```diff
@@ -0,0 +1,55 @@
+import { StyleSheet, } from 'react-native'
+import { screenSize, } from '../../utils'
+
+const styles = StyleSheet.create({
+  container: {
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: 'rgba(0, 0, 0, 0.7)',
+  },
+  popupBlock: {
+    width: screenSize.width - 40,
```

Comment:
Get rid of magic numbers in calculations:

Example:
`width: screenSize.width - STATUS_BAR_HEIGHT`

Reply:
Agree